.github/workflows/release.yml

```diff
@@ -4,22 +4,20 @@ on:
   push:
     tags:
       - "v*.*.*"
+    branches:
+      - main
+      - test/release
 
 permissions:
   contents: read
 
 jobs:
-  test:
```

Comment:
is it possible to keep this `if: startsWith(github.ref, 'refs/tags')` ?

Reply:
Ups... I should not commit this change at all